docs/notes.md

```diff
@@ -603,4 +603,254 @@ sends a pair \\((L\_j,R\_j)\\) of points at each step \\(j = k\dots1\\). An
 additional and final step involves sending a pair of scalars
 \\((a^{(0)}\_0,b^{(0)}\_0)\\) and checking the final relation directly.
 
-[bulletproofs_paper]: https://eprint.iacr.org/2017/1066.pdf
+Aggregated Range Proof
+======================
+
+We want to create an aggregated range proof for `m` values that is more efficient to create and verify than `m` individual range proofs.
```

Comment:
nit: `` `m` `` should be `\\(m\\)` so that it's math instead of code

Reply:
👍